src/content/docs/errors-inbox/legacyerrors-eol.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: 'Errors inbox transition'
+metaDescription: "The errors inbox is transitioning to an improved errors experience starting on January 31st 2023."
+freshnessValidatedDate: never
+---
+
+We're excited to introduce a new and improved errors experience, designed to provide a more comprehensive and intuitive way to manage errors in your applications. To ensure a smooth transition, please take note of the following important updates:
+
+## Important Dates:
+
+* **January 31, 2024:** The legacy errors experience will be discontinued.
+* **Switch Now!** Access the new experience today by selecting the Show new view option.
```

Comment:
```suggestion
* **Switch Now!** Access the new experience today by selecting the **Show new view** option.
```

Reply:
This shouldn't be a bullet point, and should have an accompanying screenshot + UI instructions.